{"name": "test_unsuccessfull_accept_order_without_id_order", "status": "failed", "statusDetails": {"message": "AssertionError: assert 404 == 400\n +  where 404 = <Response [404]>.status_code", "trace": "self = <tests.test_accept_order.TestAcceptOrder object at 0x000002AE1F3C2C40>, create_courier = <src.models.users.Courier object at 0x000002AE1F543B60>\n\n    def test_unsuccessfull_accept_order_without_id_order(self, create_courier):\n        with allure.step(\"формирование тела запроса\"):\n                payload_courier = {\n                    \"login\": create_courier.login,\n                    \"password\": create_courier.password\n                }\n    \n        with allure.step(\"отправляем запрос на логин и сохраняем ответ в переменную response_login\"):\n            response_login = requests.post(Path.LOGIN, data=payload_courier)\n        with allure.step(\"проверка кода и тела ответа\"):\n            create_courier.user_id = response_login.json()[\"id\"]\n            assert response_login.status_code == 200\n        with allure.step(\"отправляем запрос на логин и сохраняем ответ в переменную response_accept_order\"):\n            params = {\"courierId\": create_courier.user_id}\n            response_accept_order = requests.put(Path.ORDER_ACCEPT, params=params)\n        with allure.step(\"проверка кода и тела ответа\"):\n>           assert response_accept_order.status_code == 400\nE           assert 404 == 400\nE            +  where 404 = <Response [404]>.status_code\n\ntests\\test_accept_order.py:67: AssertionError"}, "steps": [{"name": "формирование тела запроса", "status": "passed", "start": 1757435777326, "stop": 1757435777326}, {"name": "отправляем запрос на логин и сохраняем ответ в переменную response_login", "status": "passed", "start": 1757435777326, "stop": 1757435777639}, {"name": "проверка кода и тела ответа", "status": "passed", "start": 1757435777639, "stop": 1757435777639}, {"name": "отправляем запрос на логин и сохраняем ответ в переменную response_accept_order", "status": "passed", "start": 1757435777639, "stop": 1757435777937}, {"name": "проверка кода и тела ответа", "status": "failed", "statusDetails": {"message": "AssertionError: assert 404 == 400\n +  where 404 = <Response [404]>.status_code\n", "trace": "  File \"c:\\Users\\Dmitry\\Desktop\\Python YP\\Sprint_7\\tests\\test_accept_order.py\", line 67, in test_unsuccessfull_accept_order_without_id_order\n    assert response_accept_order.status_code == 400\n"}, "start": 1757435777937, "stop": 1757435777938}], "start": 1757435777326, "stop": 1757435777938, "uuid": "d9a9ada1-f6b2-4607-ad04-5fe6177a1a5e", "historyId": "9f32f0c8f06473bd1d695a47b1b9d68e", "testCaseId": "9f32f0c8f06473bd1d695a47b1b9d68e", "fullName": "tests.test_accept_order.TestAcceptOrder#test_unsuccessfull_accept_order_without_id_order", "labels": [{"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_accept_order"}, {"name": "subSuite", "value": "TestAcceptOrder"}, {"name": "host", "value": "DESKTOP-A445VSF"}, {"name": "thread", "value": "9672-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_accept_order"}], "titlePath": ["tests", "test_accept_order.py", "TestAcceptOrder"]}